components/service/firefox-accounts/src/main/java/mozilla/components/service/fxa/sync/SyncManager.kt

```diff
@@ -110,7 +110,7 @@ object GlobalSyncableStoreProvider {
  */
 internal interface SyncDispatcher : Closeable, Observable<SyncStatusObserver> {
     fun isSyncActive(): Boolean
-    fun syncNow(reason: SyncReason, debounce: Boolean = false)
+    fun syncNow(reason: SyncReason, debounce: Boolean = false, engines: List<SyncEngine> = listOf())
```

Comment:
This probably should continue to mimic `syncNow` API, e.g. something like `customEngineList: List<SyncEngine>? = null` where `null` is "do the default thing".

Reply:
If using null to explicitly denote optionality is a well established pattern, happy to update this. However, IMO empty lists still follow the [null object pattern](https://en.wikipedia.org/wiki/Null_object_pattern) and avoid messy safe unwrap syntax. 

Or was this just about keeping the names in sync? 😅

Reply:
Mostly just about keeping the names in sync. You'll either end up doing a null check, or an isEmpty check, so I'm not sure either is better here - as long as the API (even if its a private one) is clear on what the intention is, I'm happy.